src/index.ts

```diff
@@ -22,6 +23,7 @@ const listrTasks = [
   nodeModulesIgnoredFromGit,
   noExtraneousFilesCommittedToGit,
   noSecretsCommittedToGit,
+  linkOnGithubAbout,
```

Comment:
This is an asynchronous check - should be in the "Async" block below.

Reply:
sorry, for that i didnt read i just added to the list. now i know